cmd/nerdctl/run_cgroup_linux.go

```diff
@@ -376,3 +368,18 @@ func withCustomMemoryResources(memoryOptions customMemoryOptions) oci.SpecOpts {
 		return nil
 	}
 }
+
+func checkSupportBlockIOWeight() bool {
+	if infoutil.CgroupsVersion() == "1" {
```

Comment:
cgroup v1 blkio weight was removed in kernel 5.0

https://github.com/torvalds/linux/commit/f382fb0bcef4c37dc049e9f6963e3baf204d815c